include/opt-sched/Scheduler/array_ref2d.h

```diff
@@ -102,4 +102,4 @@ template <typename T> class MutableArrayRef2D : public ArrayRef2D<T> {
 } // namespace opt_sched
 } // namespace llvm
 
-#endif
+#endif
```

Comment:
Is your editor removing newlines from the end of files? It's preferred that there is a single newline at the end of each file, so I'd ask that the newlines be added back.